File: app/src/main/java/com/arny/flightlogbook/models/Type.java
package com.arny.flightlogbook.models;

import org.chalup.microorm.annotations.Column;
public class Type {
	@Column("type_id")
	private int typeId;
	@Column("airplane_type")
	private String typeName;

	public int getTypeId() {
		return typeId;
	}

	public void setTypeId(int typeId) {
		this.typeId = typeId;
	}

	public String getTypeName() {
		return typeName;
	}

	public void setTypeName(String typeName) {
		this.typeName = typeName;
	}
}

File: app/src/main/java/com/arny/flightlogbook/data/models/Type.kt
package com.arny.flightlogbook.data.models

import com.arny.arnylib.utils.Utility
import org.chalup.microorm.annotations.Column

class Type {
    @Column("type_id")
    var typeId: Int = 0
    @Column("airplane_type")
    var typeName: String? = null

    override fun toString(): String {
        return Utility.getFields(this)
    }
}
